### PR TITLE
feat: KAN-403 - Eslint shareable config

### PR DESCRIPTION
**Pull request by andriiholubenko**

### CODE REVIEW

File: .changeset/bright-onions-shake.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@astrumu/eslint-config": minor
+---
+
+KAN-403 CI/CD process for shared-workflows repository / KAN-393 Create ESlint shareable config
```
Comment (vtereshyn) [inline .changeset/bright-onions-shake.md:5]:
```suggestion
KAN-403 - CI/CD process for shared-workflows repository / KAN-393 Create ESlint shareable config
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "@changesets/cli": "^2.26.1",
+    "@changesets/get-release-plan": "^3.0.8",
+    "@changesets/changelog-github": "^0.4.8",
+    "rimraf": "3.0.2"
+  },
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "repository": "AstrumU/shared-workflows",
+  "author": "andriiholubenko <andrii@astrumu.com>"
```
Comment (vtereshyn) [inline package.json:26]: please, remove author

File: package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "packages/**"
+  ],
+  "scripts": {
+    "clean": "pnpm -r exec rimraf {node_modules,dist,tsconfig.tsbuildinfo}",
+    "release": "changeset publish"
+  },
+  "devDependencies": {
+    "@changesets/cli": "^2.26.1",
+    "@changesets/get-release-plan": "^3.0.8",
+    "@changesets/changelog-github": "^0.4.8",
```
Comment (vtereshyn) [inline package.json:19]:
```suggestion
    "@changesets/cli": "2.26.1",
    "@changesets/get-release-plan": "3.0.8",
    "@changesets/changelog-github": "0.4.8",
```

File: packages/eslint-config/src/json.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        extends: ["plugin:jsonc/recommended-with-json"],
+        rules: {
+          "jsonc/array-element-newline": ["error", "consistent"],
+          "jsonc/array-bracket-newline": ["error", "consistent"],
+          "jsonc/indent": ["error", 2],
+          "no-multiple-empty-lines": "error",
+        },
+      },
+    ],
+  };
```
Comment (vtereshyn) [inline packages/eslint-config/src/json.js:16]:
```suggestion
  };
 
```

File: packages/eslint-config/src/react.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  },
+  extends: ["plugin:react/recommended"],
+  plugins: ["react", "react-hooks"],
+  env: {
+    es6: true,
+    browser: true,
+    node: true,
+  },
+  rules: {
+    /* react */
```
Comment (vtereshyn) [inline packages/eslint-config/src/react.js:22]:
```suggestion
```

File: .github/workflows/wf-vars.env [hunk 6]
```diff
@@ -0,0 +1,3 @@
+PACKAGE_MANAGER=pnpm
+CACHE_VERSION=v1
```
Comment (vtereshyn) [inline .github/workflows/wf-vars.env:2]: should it have prefix like we have in cdo, txp?

File: .gitignore [hunk 7]
```diff
@@ -1,3 +1,4 @@
 node_modules
 .pnpm-store
-pnpm-lock.yaml
+
+!wf-vars.env
```
Comment (vtereshyn) [inline .gitignore:4]:
```suggestion
!wf-vars.env

```

**Review by vtereshyn — CHANGES_REQUESTED** [review]

File: .github/workflows/wf-vars.env (hunk 6 above)
Comment (andriiholubenko) [reply .github/workflows/wf-vars.env:2]: this is not a required value, since the cache is stored in each repository separately, not in the organization.

**Pull request merged**